pubspec.yaml

```diff
@@ -9,12 +9,12 @@ homepage: https://aeyrium.com/
 repository: https://github.com/aeyrium/bezier-chart
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
we should allow use this package from 2.1.0 sdk version

Reply:
Ok, I'll downgrade the sdk version.